include/class.mailer.php

```diff
@@ -523,8 +523,14 @@ function($match) use ($domain, $mime, $self) {
         $mail = mail::factory('mail');
         // Ensure the To: header is properly encoded.
         $to = $headers['To'];
-        return PEAR::isError($mail->send($to, $headers, $body))?false:$messageId;
-
+        $result = $mail->send($to, $headers, $body);
+        if(!PEAR::isError($result))
+            return $messageId;
+
+        $alert=sprintf(__("Unable to email via php mail function:%1\$s\n\n%2\$s\n"),
```

Comment:
A note on the translation functions: `_S` is used to translate to the system primary language, `__` translates to the language of the request. They might be different, and messages logged to the system log should be in the primary language.

Reviewing the code in the `::send()` method currently, it looks like we have this issue in several other places.